AWS/DirectoryInsights/get-jcdirectoryinsights.py

```diff
@@ -1,4 +1,4 @@
-import requests, datetime, json, boto3, os, gzip
+import requests, datetime, json, boto3, os, gzip, logging
```

Comment:
In addition to installing boto3 do we need to deploy with the logging libraries ex:

`~/jc-directoryinsights$ pip3 install logging -t .`

Or is the logging library something that's included by default?

Reply:
Included, no need to do anything.